ls1intum/Apollon_standalone
# `Development`: Update dependencies and remove inline importing of bootstrap

### Motivation and Context
This PR updates the dependencies of Apollon Standalone and removes the inline import of bootstrap to the project.
We now use bootstrap imported from dependencies instead.


### Steps for Testing
1. Deploy the branch `chores/update-dependencies-fix-bootstrap` to the test server via bamboo
2. Once its deployed, goto [test server](https://test1.apollon.ase.in.tum.de/)
3. Check that there are no issues with the application (styling and functionality)
